3rdParty/libfmt/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+if(NOT DEVILUTIONX_SYSTEM_LIBFMT)
+  include(FetchContent_MakeAvailableExcludeFromAll)
+
+  add_definitions(-D_POSIX_C_SOURCE=200809L)
+
+  if(DEVILUTIONX_STATIC_LIBFMT)
+    set(BUILD_SHARED_LIBS OFF)
+  else()
+    set(BUILD_SHARED_LIBS ON)
+  endif()
+  include(FetchContent)
+  FetchContent_Declare(fmt
+      GIT_REPOSITORY https://github.com/fmtlib/fmt.git
+      GIT_TAG 7bdf0628b1276379886c7f6dda2cef2b3b374f0b
```

Looks like fmt does not use git submodules. In that case, it's a lot faster to download a zip archive:

```suggestion
        URL https://github.com/fmtlib/fmt/archive/7bdf0628b1276379886c7f6dda2cef2b3b374f0b.zip
        URL_HASH MD5=70e0a0beda4bdac7debe60d380e1d802
```